### PR TITLE
[ADD][11.0] sale_report_supplier_filter: New module

### PR DESCRIPTION
**Pull request by chienandalu**
This module allows to filter by product supplier in the sale reports to obtain
the sales for that specific set of products.

cc @Tecnativa TT21143

### CODE REVIEW

**Comment by rafaelbn** [comment]
@chienandalu please could you check travis? runbot is not available. Thanks! looks great! :smile:

**Comment by pedrobaeza** [comment]
Please don't use GH conflict resolution or add merge commits as this mess up commit history.

**Comment by chienandalu** [comment]
Sorry, I pushed before rebase

File: sale_report_supplier_filter/reports/report_all_channels_sales.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ReportAllChannelsSales(models.Model):
+    _inherit = "report.all.channels.sales"
+
+    seller_id = fields.Many2one(
+        related='product_id.seller_ids.name',
```
Comment (rousseldenis) [inline sale_report_supplier_filter/reports/report_all_channels_sales.py:10]: IMHO, you have to say in README that this is applicable for main supplier at time of report, if the main supplier was another one at sale time, the result could be incorrect. Or I misunderstand the aim of this?

**Review by rafaelbn — APPROVED** [review]
:+1:  tested in production

File: sale_report_supplier_filter/reports/report_all_channels_sales.py (hunk 1 above)
Comment (rafaelbn) [reply sale_report_supplier_filter/reports/report_all_channels_sales.py]: Hello @rousseldenis , it shows of **any** supplier , nor the main. It is intended and discussed with different customers. All customer ask for the same thing, **any**.

**Review by carlosdauden — APPROVED** [review]
It is very useful because it allows you to filter by any provider that is established in the product supplierinfo, but as @rousseldenis says, you can comment in the README that if the field is shown, only the first in the list (at  time of report) will appear.

**Pull request merged**